eve/workers/pod-builder/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos/7
+
+COPY requirements-eve.txt /tmp/
+
+RUN yum install -y \
+    python2-pip \
+    genisoimage \
+    git \
+    && curl -fsSL https://get.docker.com/ | sh
+    && adduser --home /home/eve eve \
+    && usermod -aG docker eve
+    && pip2 install -r /tmp/requirements-eve.txt \
+    && pip2 install buildbot-worker==0.9.0.post1
```

Comment:
please bump the buildbot worker version here to at least `0.9.12`, 

Also eve build the image using a buildarg with buildbot's version so you can use this to always be "up to date" with eve.
`ARG BUILDBOT_VERSION=0.9.12`